.github/workflows/test_iree.yml

```diff
@@ -157,7 +161,23 @@ jobs:
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
 
-      - name: "Running real weights SDXL tests"
+      - name: "Running real weights SDXL prompt encoder tests"
+        id: prompt_encode
+        if: ${{ !cancelled() }}
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest iree_tests/pytorch/models/sdxl-prompt-encoder-tank \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --log-cli-level=info \
+            --timeout=1200 \
+            --durations=0 \
+            --config-files=${SDXL_PROMPT_ENCODER_CONFIG_FILE_PATH}
```

Comment:
this should be forwarded from the `matrix` through `env` like the others

```yml
    env:
      SDXL_CONFIG_FILE_PATH: iree_tests/configs/${{ matrix.sdxl-config-file }}
```

Reply:
Yeah just noticed that and updated, thanks